docs/using-the-snowplow-console/managing-console-api-authentication/index.md

```diff
@@ -55,3 +53,46 @@ You may then use this access token value to supply authorization headers for sub
 ```bash
 curl --header 'Authorization: Bearer <JWT>'
 ```
+
+Authenticating with v2 only required the API key secret. While this method and the respective credentials will continue to work, we strongly advise you to use v3 detailed below.
+
+## Credentials UI v3
+
+The following view is available for all customers under [BDP Console settings](https://console.snowplowanalytics.com/credentials):
+
+![](images/accessing-generated-api-keys.png)
+
+API keys generation view
+
+It is possible to have multiple different keys, and deletion of a key is also possible. When a new API key is created, the following view shows up:
```

Comment:
You can create multiple API keys, and it's also possible to delete any key. When a new API key is generated, the following view will appear: